### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by byndcivilization**

### CODE REVIEW

File: server/url_shortener.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+			return make_response('You must submit a url!', 400)
+
+		self._validate_url(url)
+		url_id = self._shorten_url(url)
+		return make_response(url_id)
+
+	@staticmethod
+	def _validate_url(url):
+		parts = urlparse.urlparse(url)
+		if not parts.scheme in ('http', 'https'):
```
Comment (devinmcginty) [inline server/url_shortener.py]: This is nitpicky, but I prefer the syntax `if parts.scheme not in ('http', 'https')`. It reads a bit more like english.

File: app.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+from server.health_check import HealthCheck
+from server.url_shortener import UrlShortener
+from server.url_lookup import UrlLookup
+from server.url_hit_count import UrlHitCount
+
+def create_app():
+	app = Flask(__name__)
+	app.debug = True
+
+	logging.basicConfig(level=logging.INFO,
```
Comment (devinmcginty) [inline app.py]: okay, here you're mixing spaces and tabs, which makes it look like this in github. say no to tabs.
image

File: server/url_shortener.py (hunk 1 above)
Comment (byndcivilization) [reply server/url_shortener.py]: checks out

**Comment by byndcivilization** [comment]
counterpoint: tabs are great

**Comment by byndcivilization** [comment]
Fine spaces it is

**Pull request merged**